app/scripts/views/settings/account_recovery/recovery_key.js

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const $ = require('jquery');
+const Cocktail = require('cocktail');
+const BaseView = require('../../base');
+const ModalSettingsPanelMixin = require('../../mixins/modal-settings-panel-mixin');
+const Template = require('templates/settings/account_recovery/recovery_key.mustache');
+const RecoveryKeyPrintTemplate = require('templates/settings/account_recovery/recovery_key_print.mustache');
+
+const {t} = BaseView;
+
+const View = BaseView.extend({
+  template: Template,
+  className: 'account-recovery-key',
+  viewName: 'settings.account-recovery.recovery-key',
+
+  events: {
+    'click .done-link': '_done',
+    'click .download-key': '_downloadKey',
+    'click .print-key': '_printKey'
+  },
+
+  _getFormatedRecoveryKeyFilename() {
```

Comment:
I thought about extracting print and download into their own mixin but the code churn was a bit much. We should consider this as a followup issue.